### ISSUE TITLE
Number of speakers in CN-Celeb

### ISSUE THREAD

**Issue by 12-zhx** [herbertLJY/EDITnet#1]
How do you choose 800 speakers in CN-Celeb for the project?

**Comment by herbertLJY** [comment]
Yes, 800 speakers are used in the unsupervised training of the EDITnet.

**Comment by herbertLJY** [comment]
> How do you choose 800 speakers in CN-Celeb for the project?

**Comment by 12-zhx** [comment]
Sorry, I want to know how do you select 800 speakers from 1000 speakers in CN-Celeb dataset. Which principle is used to divide the dataset named CN-Celeb.

**Comment by herbertLJY** [comment]
> Sorry, I want to know how do you select 800 speakers from 1000 speakers in CN-Celeb dataset. Which principle is used to divide the dataset named CN-Celeb.

We just follow the training set and test set on this paper :CN-Celeb 
https://arxiv.org/pdf/1911.01799v1.pdf
